src/cascadia/TerminalApp/AppCommandlineArgs.cpp

```diff
@@ -398,6 +399,54 @@ void AppCommandlineArgs::_buildMoveFocusParser()
     setupSubcommand(_moveFocusShort);
 }
 
+// Method Description:
+// - Adds the `move-pane` subcommand and related options to the commandline parser.
+// - Additionally adds the `mp` subcommand, which is just a shortened version of `move-pane`
+// Arguments:
+// - <none>
+// Return Value:
+// - <none>
+void AppCommandlineArgs::_buildMovePaneParser()

```

Comment:
This absolutely was just copied from MoveFocusParser. However, without using `#define`s I'm not sure how it could be abstracted because it relies on class members and types that are different for each. It isn't much of an abstraction if I have to pass in 3 different string arguments, pointers to some locals, and also have it parameterized by the argument type.